<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGCSwarmRemote</class>
 <widget class="QWidget" name="QGCSwarmRemote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="CheckAll">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Check All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearAll">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>211</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
